cmd/antrea-agent/agent.go

```diff
@@ -848,6 +851,21 @@ func run(o *Options) error {
 		go mcStrechedNetworkPolicyController.Run(stopCh)
 	}
 
+	klog.InfoS("Waiting for flow restoration to complete")
+	flowRestoreCompleteWait.Wait()
+	if err := agentInitializer.FlowRestoreComplete(); err != nil {
+		return err
+	}
+	klog.InfoS("Flow restoration complete")
```

Comment:
nit: s/complete/completed/ ?

Reply:
I think "is complete" or "has completed" are the best options here. I will update to the first one.